tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -89,19 +89,9 @@ sub post_fail_hook {
 
     $self->SUPER::post_fail_hook;
 
-    #Restart libvirtd service
-    # Note: TBD for modular libvirt. See poo#129086 for detail.
-    virt_autotest::utils::restart_libvirtd() if is_monolithic_libvirtd;
-
     #Destroy created virtual networks
     virt_autotest::virtual_network_utils::destroy_vir_network();
 
-    #Restore br123 for virt_autotest
-    virt_autotest::virtual_network_utils::restore_standalone();
-
-    #Restore Guest systems
-    virt_autotest::virtual_network_utils::restore_guests();
```

Comment:
Why removing all above? Will it decrease the test stability? Such change seems to be  out of the scope of modular libvirt adaption. So does similar removal in other virtual network modules' post_fail_hook.

Reply:
With a deeper understanding of modular libvirt daemons and monolithic libvirt daemon, it is advisable to minimize the frequency of restarting libvirt during our daily virtualized network functional testing. This is particularly important when encountering new errors in virtualized network function tests, as it is crucial to protect the affected site in order to quickly identify the root cause of the issue. Restarting the modular libvirt daemons or monolithic libvirt daemon may introduce new problems and hinder the troubleshooting process. 
Meanwhile, this code changes was so helpful during local validation run about virtualized network functional testing.

Reply:
okay, if it is well verified , no harm but improvement, I will be happy with it